packages/components/tests/dummy/app/templates/components/form/radio-card.hbs

```diff
@@ -0,0 +1,625 @@
+{{page-title "Form::RadioCard Component"}}
+
+<h2 class="dummy-h2">Form::RadioCard</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">The
+    <code class="code-dummy">RadioCard</code>
+    is an input of type "radio" with a bigger target area represented by a card container. Same as the
+    <LinkTo @route="components.form.radio"><code class="code-dummy">Form::Radio</code></LinkTo>
+    component, the
+    <code class="code-dummy">RadioCard</code>
+    ​​ is a form element that allows users to select a single item from a list of related options.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <h4 class="dummy-h4">Form::RadioCard</h4>
+  <p class="dummy-paragraph" id="component-api-form-radio-card">
+    Here is the API for the
+    <code class="code-dummy">RadioCard</code>
+    component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-form-radio-card">
```

Comment:
I _think_ from this APIs the following arguments/props are missing:
- `@controlPosition`
- `@align`